tgui/packages/tgui/interfaces/NtosNotepad.tsx

```diff
@@ -371,32 +368,21 @@ type RetryActionType = (retrying?: boolean) => void;
 export const NtosNotepad = (props) => {
   const { act, data, config } = useBackend<NoteData>();
   const { note } = data;
-  const [documentName, setDocumentName] = useLocalState<string>(
-    'documentName',
-    DEFAULT_DOCUMENT_NAME,
-  );
-  const [originalText, setOriginalText] = useLocalState<string>(
-    'originalText',
-    note,
+  const [documentName, setDocumentName] = useState<string>(
+    DEFAULT_DOCUMENT_NAME
   );
+  const [originalText, setOriginalText] = useState<string>(note);
```

Comment:
```suggestion
  const [documentName, setDocumentName] = useState(
    DEFAULT_DOCUMENT_NAME
  );
  const [originalText, setOriginalText] = useState(note);
```
these are all inferred